api/v1/db/oracledb/game-dao.js

```diff
@@ -177,6 +179,43 @@ const cleanTableCardsByGameId = async (gameId, connection) => {
   await connection.execute(cleanCardSqlQuery, sqlParams);
 };
 
+const insertCardsByGameId = async (gameId, tableCards, connection) => {
+  /**
+   * The code below flattens the array of suit-number pair of every single card object of
+   * tableCards. The flattened array will be used to generate bind strings so only one
+   * database call is needed to insert all the cards in the tableCards array.
+   */
```

Comment:
Nice job on documenting this. Could you change `/**` to `/*` at the top? This will make sure the comment is a general comment instead of a JSDoc comment, since it applies to more than just  `flattenedArray`.